pyt/__main__.py

```diff
@@ -30,7 +30,27 @@
 )
 
 
-def main(command_line_args=sys.argv[1:]):  # noqa: C901
+def discover_files(targets, excluded_files, recursive=False):
+    included_files = list()
+    excluded_list = excluded_files.split(",")
+    
+    
+    for target in targets:
+        if os.path.isdir(target):
+                for root, dirs, files in os.walk(target):
+                    for f in files:
+                        if not recursive:
```

Comment:
It's important for the `if not recursive:` to be after the

```python
                        fullpath = os.path.join(root, f)
                        if os.path.splitext(fullpath)[1] == '.py' and fullpath.split("/")[-1] not in excluded_list:
                            included_files.append(fullpath)
```

this is so that we only iterate through the `for f in files:` once. i.e. just one-level of depth and not recursively.

Reply:
Ok, i did it